src/java/utils/JsonHelper.java

```diff
@@ -288,11 +289,13 @@ public static Map<String, Object> buildPage(Folio f) throws SQLException, IOExce
     */
      
      
-    public static Map<String, Object> buildPage(int projID, String projName, Folio f, User u, String profile) throws SQLException
-   {
+//    public static Map<String, Object> buildPage(int projID, String projName, Folio f, User u, String profile) throws SQLException
+      public static JSONObject buildPage(int projID, String projName, Folio f, User u, Map services, String profile) throws SQLException
+	{
 	try 	{
                 System.out.println("v3 buildpage");
```

Comment:
Either remove this or add more information for logging, like the project, though this will get called a lot.